tests/readme.md

```diff
@@ -86,3 +93,6 @@ Options include:
 - `PLUGINS_TEST_CLI_PATH` specifies an alternative path to a trunk binary
 - `PLUGINS_TEST_LINTER_VERSION` specifies a linter version semantic (KnownGoodVersion | Latest |
```

Comment:
Now that I actually read this... this is a bit confusing, because this propagates to every single linter test running. In other words, if I do

```
PLUGINS_TEST_LINTER_VERSION=1.4.4 npm test
```

It applies 1.4.4 to every linter we're testing, and not just sqlfluff.

I think if we use a schema where we use snapshot names to define the test cases, that will solve this problem.

Reply:
Not sure I follow. Let's discuss